src/Service/CognitoIdentityProvider/tests/Unit/Result/AdminConfirmSignUpResponseTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace AsyncAws\CognitoIdentityProvider\Tests\Unit\Result;
+
+use AsyncAws\CognitoIdentityProvider\CognitoIdentityProviderClient;
+use AsyncAws\CognitoIdentityProvider\Input\AdminConfirmSignUpRequest;
+use AsyncAws\CognitoIdentityProvider\Result\AdminConfirmSignUpResponse;
+use AsyncAws\Core\Response;
+use AsyncAws\Core\Test\Http\SimpleMockedResponse;
+use AsyncAws\Core\Test\TestCase;
+use Psr\Log\NullLogger;
+use Symfony\Component\HttpClient\MockHttpClient;
+
+class AdminConfirmSignUpResponseTest extends TestCase
+{
+    public function testAdminConfirmSignUpResponse(): void
+    {
+        // see https://docs.aws.amazon.com/cognitoidentityprovider/latest/APIReference/API_AdminConfirmSignUp.html
+        $response = new SimpleMockedResponse('{
+        }');
+
+        $client = new MockHttpClient($response);
+        $result = new AdminConfirmSignUpResponse(new Response($client->request('POST', 'http://localhost'), $client, new NullLogger()));
+        self::assertTrue($result->resolve());
```

Comment:
Not sure how to test an empty response !

Reply:
https://docs.aws.amazon.com/aws-sdk-php/v3/api/api-cognito-idp-2016-04-18.html#adminconfirmsignup

Reply:
Neither am I. Let's leave it like this for now